Developer.md

```diff
@@ -0,0 +1,29 @@
+Relax Module Development
```

Comment:
@lmajano :  I've added this file (Developer.md) to the Relax repo to help clarify setup and developoment conventions